src/main/java/com/three_tech_solutions/slot_app/services/implementations/UserServiceImpl.java

```diff
@@ -31,20 +32,26 @@ public class UserServiceImpl implements UserService {
     private final StudentMapper studentMapper;
     private final PasswordEncoder passwordEncoder;
 
+
     @Override
     public User loadUserByUsername(String username) throws UsernameNotFoundException {
         return userRepository.findByUsername(username)
                 .orElseThrow(() -> new ResponseStatusException(BAD_REQUEST, "Credenciales ingresadas incorrectas"));
     }
 
     @Override
-    public Page<StudentResponse> getUserStudents(UUID userId, String filter, Pageable pageable) {
-        return studentService.getStudentsByUserAndNameAndLastNameAndDni(
+    public Page<StudentResponse> getUserStudents(UUID userId, String filter, String sortBy, String sortDirection, Pageable pageable) {
+        Page <Student> students =studentService.getStudentsByUserAndNameAndLastNameAndDni(
                     getUserByIdOrThrowException(userId),
                     filter,
+                    sortBy,
+                    sortDirection,
                     pageable
-            )
+            );
+                return students
                 .map(studentMapper::toStudentResponse);
+
+
     }
```

Comment:
Esto por que se modifico? Podes dejarlo como estaba antes

```suggestion
    public Page<StudentResponse> getUserStudents(UUID userId, String filter, String sortBy, String sortDirection, Pageable pageable) {
        return studentService.getStudentsByUserAndNameAndLastNameAndDni(
                    getUserByIdOrThrowException(userId),
                    filter,
                    sortBy,
                    sortDirection,
                    pageable
            )
            .map(studentMapper::toStudentResponse);
}
```